tutorials/tutorial2.ipynb

```diff
@@ -496,18 +369,20 @@
    ],
    "source": [
     "!atacworks denoise \\\n",
-    "    --input_files NK.50_cells.h5 \\\n",
+    "    --noisybw dsc.1.NK.50.cutsites.smoothed.200.bw \\\n",
```

Comment:
Same questions - config files should be supplied, why not use `--distributed`?

Reply:
No need of using --config for tutorial. Default configs/infer_config.yaml params should be sufficient.